components/02-molecules/callout/callout.yml

```diff
@@ -1,5 +1,5 @@
 callout__heading: 'Degree Programs'
-callout__text: 'Designed for those who intend to pursue graduate and those who wish to immediately enter a career in which broad scientific training is beneficial.'
+callout__text: '<p>Designed for those who intend to pursue graduate and those who wish to immediately enter a career in which broad scientific training is beneficial.</p><p>Designed for those who intend to pursue graduate and those who wish to immediately enter a career in which broad scientific training is beneficial.</p>'
```

Comment:
I don't think Ashley will want this demoed with two paragraphs. It's good to see it working that way, but let's remove it from the Storybook example so that people don't think that they're supposed to do that.

Reply:
@ModulesUnraveled I removed the extra `p` element